docker-qgis/utils.py

```diff
@@ -366,18 +404,6 @@ def eval(self, root: Path) -> Path:
         return path
 
 
-class BaseException(Exception):
```

Comment:
Just double check that `BaseException` is no longer needed.

Reply:
Renamed to "QfcException" already.